addons/common/XEH_preStart.sqf

```diff
@@ -0,0 +1,31 @@
+#include "script_component.hpp"
+
+//See usage in XEH_preInit
+private _cfgPatches = configFile >> "CfgPatches";
+private _addonsFull = ("true" configClasses _cfgPatches) apply {configName _x};
+
+LOG("addonsFull: " + str count _addonsFull);
+
+//Filter out addons that don't have any units defined as we don't need to activate these
+private _addonsSparse = _addonsFull select {
+    !(getArray(_cfgPatches >> _x >> "units") isEqualTo [])
+};
+LOG("addonsSparse1: " + str count _addonsSparse);
+
+//Filter out addons defined in CfgAddons as they are always activated
+private _cfgAddonsCategories = ("true" configClasses (configFile >> "CfgAddons"));
```

Comment:
Why the parentheses?

Reply:
Because I'm a fox and you're an angel u cocoon.